winui/Masked-TextBox/Customization.md

```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Customization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Customization in WinUI MaskedTextBox
+
+This section explains about header, header template and description properties of MaskedTextBox.
+
+## Header
+
+The `Header` property is used to display the title for the MaskedTextBox Control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedTextBox x:Name="sfMaskedTextBox"
```

Comment:
@Vignesh-SF3580 remove the unwanted variable name declaration and set width as 200.

Reply:
Removed the variable name and set width as 200.